lib/ASRouter.jsm

```diff
@@ -713,6 +714,14 @@ class _ASRouter {
     return {experiment, interrupt, triplet};
   }
 
+  // Dispatch a TRAILHEAD_ENROLL_EVENT action
+  _sendTrailheadEnrollEvent(data) {
+    this.dispatchToAS(ac.AlsoToMain({
```

Comment:
This doesn't need `AlsoToMain` just a plain action object should be sufficient as we're going from main to main.